src/test/Components/CreateImageWizard/CreateImageWizard.test.js

```diff
@@ -399,52 +400,50 @@ describe('Step Upload to AWS', () => {
     await clickNext();
     await clickNext();
 
-    const composeImage = jest
-      .spyOn(api, 'composeImage')
-      .mockImplementation((body) => {
-        expect(body).toEqual({
-          distribution: RHEL_9,
-          image_name: undefined,
-          customizations: {
-            packages: undefined,
-          },
-          image_requests: [
-            {
-              architecture: 'x86_64',
-              image_type: 'aws',
-              upload_request: {
-                type: 'aws',
-                options: {
-                  share_with_sources: ['123'],
-                },
-              },
-            },
-          ],
-        });
-        const id = 'edbae1c2-62bc-42c1-ae0c-3110ab718f5a';
-        return Promise.resolve({ id });
-      });
+    // const composeImage = jest
```

Comment:
Same question - why are we only commenting the mockImplementation? Is it to keep an example for the future?